com.unity.multiplayer.mlapi/Editor/CodeGen/ILPostProcessorProgram.cs

```diff
@@ -11,8 +11,31 @@
 
 using Assembly = System.Reflection.Assembly;
 
+using ILPPInterface = Unity.CompilationPipeline.Common.ILPostProcessing.ILPostProcessor;
+
 namespace MLAPI.Editor.CodeGen
 {
+    // There is a behaviour difference between 2019.4 and 2020+ codegen
```

Comment:
Could we #ifdef NITY_2019_OR_OLDER around all this and, safe as it seems, eliminate any exposure to 2020?  (also help us find this later to remove when we no longer want the workaround)?

Reply:
or rather `#if !UNITY_2020_3_OR_NEWER` (there is no `XXX_OR_OLDER` defines AFAIK)

Reply:
oh actually, we already have that on top of this file 😛 (line 1)